oqs-test/try_connection.py

```diff
@@ -29,46 +130,74 @@
     "ssh-oqsdefault",
     "ssh-rsa3072-oqsdefault",
     "ssh-ecdsa-nistp256-oqsdefault",
-    "ssh-dilithium2",
```

Comment:
why did Dilithium2 get removed here?

Reply:
This is code output by `generate.py` -- and that cannot create the full matrix of options due to `openssh` internal limitations as to the number of sig algs supported. So I pruned `dilithium2` as its "brethren" `dilithium3` and `dilithium2aes` are already included. If you have a specific need for `dilithium2` that surely can be re-activated by default [here](https://github.com/open-quantum-safe/openssh/blob/0e7456405ee71a775ead6094913741b2cccd9618/oqs-template/generate.yml#L478) but another algorithm should be de-activated in turn. Suggestions?

Reply:
Right. Might be good to document this in the README.

Reply:
Never mind, it's already listed in the readme, though it might be good to add some more emphasis to the "not everything is enabled" sentence and perhaps clarify that it's due to technical restrictions.

Reply:
Makes sense. So done in https://github.com/open-quantum-safe/openssh/wiki/Using-liboqs-supported-algorithms-in-the-fork#code-generation